### PR TITLE
Fix GitHub lib/pq link

### PR DESCRIPTION
**Pull request by RagnarArdal**
Old link was relative and thus didn't work.

### CODE REVIEW

**Review by bamurtaugh — DISMISSED** [review]
LGTM, thank you for the contribution!

**Comment by bamurtaugh** [comment]
@samruddhikhandale requesting your approval for the workflow run (in case there's anything you'd like to check first before we merge).

File: src/go-postgres/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ The `ports` property in `docker-compose.yml` [publishes](https://docs.docker.com
 
 ### Installing Go Dependencies
 
-This template includes the popular [PostGres Driver Library for Go](github.com/lib/pq). This is the recommended driver for use with Go, as per [GoLang Documentation](https://golangdocs.com/golang-postgresql-example).
+This template includes the popular [PostGres Driver Library for Go](https://github.com/lib/pq). This is the recommended driver for use with Go, as per [GoLang Documentation](https://golangdocs.com/golang-postgresql-example).
```
Comment (samruddhikhandale) [inline src/go-postgres/README.md:55]: Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file and update NOTES.md instead to avoid merge conflicts? Thanks!

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/go-postgres/README.md (hunk 1 above)
Comment (RagnarArdal) [reply src/go-postgres/README.md]: Thanks, I didn't realize this, so I've made the suggested changes.

**Review by samruddhikhandale — APPROVED** [review]
Thanks!
